SubjuGator/gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -116,8 +142,14 @@ def get_ranges(self) -> Tuple[np.ndarray, np.ndarray]:
         )
         return minima, maxima
 
-    def get_thruster_wrench(self, position, direction):
-        """Compute a single column of B, or the wrench created by a particular thruster"""
+    def get_thruster_wrench(self, position: list, direction: list) -> np.ndarray:
```

Comment:
For containers in Python 3, we should also attempt to type the objects that are stored in the container. Because these parameters take `list`s of `float`s, maybe we could change the annotations to `list[float]`?

Reply:
Keep in mind though that we will also need to import `from __future__ import annotations` to make that typing syntax work. You can add that to the top of the file.

Reply:
you are the boss of typing